skll/learner.py

```diff
@@ -910,7 +915,7 @@ def __init__(self, model_type, probability=False, pipeline=False,
         self.scaler = None
         self.label_dict = None
         self.label_list = None
-        self.pos_label_str = pos_label_str
+        self.pos_label_str = safe_float(pos_label_str) if pos_label_str is not None else pos_label_str
```

Comment:
And, continuing the comment from `config.py`, is the `pos_label_str` will never be None but will be `None` string coming here, so do we need `safe_float` here?

Reply:
It won't be `None` string so this does not apply.